src/constants.ts

```diff
@@ -374,6 +375,7 @@ export const BASE_R4_RESOURCES: R4Resource[] = [
     'StructureMap',
     'Subscription',
     'Substance',
+    'SubstanceNucleicAcid',
     'SubstancePolymer',
     'SubstanceProtein',
     'SubstanceReferenceInformation',

```

Comment:
could we swap line 382 and 383 (so they can be alphabetized?

'SubstanceSourceMaterial',
'SubstanceSpecification',

Reply:
done